src/package-bazel.sh

```diff
@@ -66,4 +65,15 @@ if [ -n "${EMBEDDED_TOOLS}" ]; then
   (cd ${PACKAGE_DIR}/embedded_tools && unzip -q "${WORKDIR}/${EMBEDDED_TOOLS}")
 fi
 
+# Unzip platforms.zip into platforms/, move files up from external/platforms
+# subdirectory, and cleanup after itself
+( \
```

Comment:
This looks somewhat complicated... (I'm not sure if there is a simpler way to cut the first few path segments, though)

Reply:
I was shooting for the simplest possible thing that could work. I can remove external/platforms in the genrule above creating the archive, but this looked simpler to me.

Reply:
Okie. Greenlight then.